src/docs/releases/2.0.0.md

```diff
@@ -269,7 +269,52 @@ Here are the updated signatures:
    **To:**
    `Task<IDisplayResult> UpdateAsync(TSection section, UpdateEditorContext context)`
 
-These adjustments ensure compatibility and adherence to the latest conventions within the `SectionDisplayDriver` class.
+These adjustments ensure compatibility and adherence to the latest conventions within the `DisplayDriver` class.
```

Comment:
The "To" methods mentioned under points 1 and 2 above don't exist anymore.

Reply:
In the `SectionDisplayDriver` we want people to use `UpdateAsync(TModel model, TSection section, UpdateEditorContext context)`.

In the `DisplayDriver` we want people to use `UpdateAsync(TModel model, UpdateEditorContext context)`

Both of these methods exists,

Reply:
Under point 1 the To method is `Task<IDisplayResult> UpdateAsync(TModel model, TSection section, IUpdateModel updater, UpdateEditorContext context)` in `SectionDisplayDriver`. There's no such method.

Point 2 refers to `Task<IDisplayResult> UpdateAsync(TSection section, IUpdateModel updater, UpdateEditorContext context)` in `SectionDisplayDriver`. There's no such method either.

Reply:
I think I am not understanding here.

In the `SectionDisplayDriver`, the update obsolete message states the following:

> This method is obsolete and will be removed in version 3. Instead, use the UpdateAsync(TModel model, TSection section, UpdateEditorContext context)

This method exists here:

[        public virtual Task<IDisplayResult> UpdateAsync(TModel model, TSection section, UpdateEditorContext context)](https://github.com/OrchardCMS/OrchardCore/blob/53869c4d516c997ae334fbdf2e84421298c0698f/src/OrchardCore/OrchardCore.DisplayManagement/Entities/SectionDisplayDriver.cs#L111C27-L111C117)

In the 2.0.0 release notes we list the methods that were removed like the one you mentioned in your comment:

```
 and **removed** the following methods:

- `UpdateAsync(TModel model, TSection section, IUpdateModel updater, UpdateEditorContext context)`
- `UpdateAsync(TSection section, IUpdateModel updater, UpdateEditorContext context)`
- `UpdateAsync(TSection section, IUpdateModel updater, string groupId)`
- `UpdateAsync(TSection section, UpdateEditorContext context)`
- `EditAsync(TSection section, BuildEditorContext context)`
- `DisplayAsync(TSection section, BuildDisplayContext context)`
```

What am I missing here?

Reply:
There's this section in the release notes I was referring to in my opening comment:

![image](https://github.com/user-attachments/assets/00096bac-a4f7-4c56-ad79-17f14bcb1a12)

Point 1 and 2 are now inaccurate.

Reply:
ahhh. Got it. I fixed this.